io.github.ihhub.Fheroes2.yaml

```diff
@@ -0,0 +1,38 @@
+app-id: io.github.ihhub.Fheroes2
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+finish-args:
+  - --device=all
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Pulseaudio access
+  - --socket=pulseaudio
+  # OpenGL access
+  - --device=dri
+  # SDL needs to be able to talk to UPower
+  - --system-talk-name=org.freedesktop.UPower
+  # Allow SDL to inhibit the screensaver
+  - --talk-name=org.freedesktop.ScreenSaver
```

Comment:
Similarly to OpenTyrian, I don't see why a game should have these permissions.

Reply:
done